css/main.css

```diff
@@ -3,18 +3,26 @@
 	--base-color: #E5D9B9;
 	--lighter-color: #FBEFD5;
 	--lightest-color: #FFF5B5;
-	--topbar-height: 2.5em;
 }
 
 html{
 	font-family: Calibri,serif;
-	scroll-padding-top: var(--topbar-height);/*below topbar*/
+	width: 100vw;
```

Comment:
generally, i dislike html width:100vw as it usually causes stuff to break horribly if you're not using the "right" browser and window dimensions.
It might be OK but i will be scrutinizing, and i think its possible to do this without 100vw.

Reply:
ok absolutely not possible for some pages like map.

Reply:
Muggy pointed out that the html by default fills the window without any width/height parameters, so I removed them and the appearance didn't change. Does this fix what you are saying, or would having the body by 100vw cause the same issues? I'm not sure what stuff breaks horribly to test this.